<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPlayWid</class>
 <widget class="QWidget" name="QPlayWid">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="DiscGroupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QGraphicsView" name="CoverGraphics">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>90</y>
        <width>208</width>
        <height>208</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="DiscGraphics">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>63</y>
        <width>262</width>
        <height>262</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
     <zorder>DiscGraphics</zorder>
     <zorder>CoverGraphics</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="LyricGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="NameLbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>歌曲名</string>
      </property>
     </widget>
     <widget class="QLabel" name="SingerLbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>79</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>歌手</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="LrcText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>119</y>
        <width>361</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
